<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QMainWindow" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>635</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>998</width>
    <height>635</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>998</width>
    <height>635</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_bg_Lg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1001</width>
      <height>651</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/newPrefix/image/signup.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_register">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>46</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : white;</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSUsername">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Atkinson Hyperlegible Next</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 28, 75,0.4);
border: 3px solid white;
border-radius: 10px;
color:white;
font: 75 11pt &quot;Atkinson Hyperlegible Next&quot;;
padding-left:9px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Insert Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSPass">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Atkinson Hyperlegible Next</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 28, 75,0.4);
border: 3px solid white;
border-radius: 10px;
color:white;
font: 75 11pt &quot;Atkinson Hyperlegible Next&quot;;
padding-left:9px;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Insert Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCfpass">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Atkinson Hyperlegible Next</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 28, 75,0.4);
border: 3px solid white;
border-radius: 10px;
color:white;
font: 75 11pt &quot;Atkinson Hyperlegible Next&quot;;
padding-left:9px;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_signup">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>410</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 28, 75,0.4);
border: 3px solid white;
border-radius: 10px;
color:white;
font: 75 11pt &quot;Atkinson Hyperlegible Next&quot;;

</string>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_register_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Atkinson Hyperlegible Next Extr</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : white;</string>
    </property>
    <property name="text">
     <string>Already have an account?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_signup_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>590</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 28, 75,0.4);
border: 3px solid white;
border-radius: 10px;
color:white;
font: 75 11pt &quot;Atkinson Hyperlegible Next&quot;;

</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
